tests/views/test_collection_exercise.py

```diff
@@ -73,6 +74,9 @@
 with open(f"{project_root}/test_data/survey/single_survey.json") as json_data:
     survey = json.load(json_data)
 
+with open(f"{project_root}/test_data/survey/single_survey_eq.json") as json_data:
```

Comment:
Can we avoid this? I was trying to slim them down. We have a lot of JSON files doing very little when they could use a generic one an update inline. It should a pain to update when one thing changes

Reply:
I thought I had removed this. I'll have a look.